### PR TITLE
chore: move udev rules

### PR DESCRIPTION
**Pull request by dsseng**
Ref: siderolabs/pkgs#1053

Signed-off-by: Dmitry Sharshakov <dmitry.sharshakov@siderolabs.com>

### CODE REVIEW

File: storage/iscsi-tools/open-iscsi/pkg.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ steps:
           -Dno_systemd=true \
           -Dprefix=/usr/local \
           -Discsi_sbindir=/usr/local/sbin \
-          -Drulesdir=/usr/local/etc/udev/rules.d \
+          -Drulesdir=/usr/lib/udev/rules.d \
```
Comment (frezbo) [inline storage/iscsi-tools/open-iscsi/pkg.yaml:54]: I this this is the default setting in most packages, if so I guess we can drop the config even

**Review by smira — APPROVED** [review]

File: storage/iscsi-tools/open-iscsi/pkg.yaml (hunk 1 above)
Comment (dsseng) [reply storage/iscsi-tools/open-iscsi/pkg.yaml:54]: Should I perhaps just remove this config line? And the similar path in nut-client?

Comment (frezbo) [reply storage/iscsi-tools/open-iscsi/pkg.yaml:54]: yes, let's do that as long as it installs the default rules at the right location

Comment (frezbo) [reply storage/iscsi-tools/open-iscsi/pkg.yaml:54]: we also need to bump extensions-validator

Comment (dsseng) [reply storage/iscsi-tools/open-iscsi/pkg.yaml:54]: open-iscsi defaults to `RULESDIR ?= $(etcdir)/udev/rules.d`, so for it I must keep the line

Comment (dsseng) [reply storage/iscsi-tools/open-iscsi/pkg.yaml:54]: And well, NUT also seems to default to etc

**Review by frezbo — APPROVED** [review]

**Review by frezbo — APPROVED** [review]

**Pull request merged**